client/shipping-label/views/purchase/steps/packages/package-info.js

```diff
@@ -0,0 +1,90 @@
+import React, { PropTypes } from 'react';
+import { translate as __ } from 'lib/mixins/i18n';
+import ItemInfo from './item-info';
+import NumberField from 'components/number-field';
+import FormLegend from 'components/forms/form-legend';
+import FormSelect from 'components/forms/form-select';
+import getBoxDimensions from 'lib/utils/get-box-dimensions';
+import _ from 'lodash';
+
+const renderPackageDimensions = ( dimensions, dimensionUnit ) => {
+	return `${dimensions.length} ${dimensionUnit} x ${dimensions.width} ${dimensionUnit} x ${dimensions.height} ${dimensionUnit}`;
+};
+
+const PackageInfo = ( { packageId, selected, all, dimensionUnit, weightUnit, errors, updateWeight, openItemMove, moveItem, removePackage, setPackageType } ) => {
+	if ( ! packageId ) {
+		return null;
+	}
+
+	const renderItemInfo = ( item, itemIndex ) => {
+		return (
+			<ItemInfo key={ itemIndex } item={ item } itemIndex={ itemIndex } packageId={ packageId } showRemove={ true } openItemMove={ openItemMove } moveItem={ moveItem } />
+		);
+	};
+
+	const renderPackageOption = ( box, boxId ) => {
+		const dimensions = getBoxDimensions( box );
+		return ( <option value={ boxId } key={ boxId }>{ box.name } - { renderPackageDimensions( dimensions, dimensionUnit ) }</option> );
+	};
+
+	const packageOptionChange = ( e ) => {
+		setPackageType( packageId, e.target.value );
+	};
+
+	const pckg = selected[ packageId ];
+
+	const renderPackageSelect = () => {
+		if ( ! all[ pckg.box_id ] ) {
+			return ( <div className="wcc-package__desc">Package - { renderPackageDimensions( pckg, dimensionUnit ) }</div> );
+		}
+
+		const packageOptions = Object.values( _.mapValues( all, renderPackageOption ) );
```

Comment:
You can use `_.map()` here instead of wrapping `_.mapValues()` with `Object.values()`.